hist/hist/test/test_tf2.cxx

```diff
@@ -0,0 +1,28 @@
+#include "TF2.h"
+
+#include "gtest/gtest.h"
+
+TEST(TF2, Save)
+{
+   TF2 linear("linear", "x+y", -10, 10, -10, 10);
+   linear.SetNpx(20);
+   linear.SetNpy(20);
+
+   // this works properly since a while
+   linear.Save(-10, 10, -10, 10, 0, 0);
+   for (Double_t x = -10.; x < 10.; x += 0.33) {
+      for (Double_t y = -10.; y < 10.; y += 0.44) {
```

Comment:
Is there a few iteration that ends 'exactly' on the boundary?

Reply:
It is not important. One tests interpolation.